src/archive/mod.rs

```diff
@@ -366,7 +398,13 @@ impl<'a> Archive<'a> {
             let name = member.raw_name();
             if name == INDEX_NAME {
                 let data: &[u8] = buffer.pread_with(member.offset as usize, member.size())?;
-                index = Index::parse_sysv_index(data)?;
+                index = if sysv_symbol_index_seen {
+                    // Second symbol index is Microsoft's extension of SysV format
+                    Index::parse_windows_linker_member(data)?
```

Comment:
Accordingly to Microsoft's documentation, the first symbol index entry (First Linker Member) is not used by linkers and is kept for compatibility reasons. Given this, it seems reasonable to simply replace the index parsed previously rather than providing access to both.

Reply:
Sounds good to me.